File: javascript/gpt/p104.js
// p104.js - write important_config and set 0700
const fs = require('fs');
const path = 'important_config';
fs.writeFileSync(path, 'important_config', { encoding: 'utf8' });
try { fs.chmodSync(path, 0o700); } catch (e) { console.error('chmod failed', e); }
console.log(path);

File: python/gpt/p104.py
"""
p104.py
Open/create important_config with mode 0700 and write 'important_config'.
"""
import os

fname = 'important_config'
# create/write
with open(fname, 'w', encoding='utf-8') as f:
    f.write('important_config')
# set to rwx------
os.chmod(fname, 0o700)
print(fname)
